test/specs/test1.js

```diff
@@ -0,0 +1,33 @@
+import { expect, browser, $ } from '@wdio/globals';
```

Comment:
Lets name this file in more informative way :)

Reply:
instead of test1.js

Reply:
Done.